### DISCUSSION TITLE
Welcome to DMD2 Discussions!

### DISCUSSION THREAD

**Discussion by johnkeel-thork** [johnkeel-thork/DMD2#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raschaa** [comment]
re: Translations

"As for terms like "Roadbook" and "GPX" / "Waypoint" / "Track" I would also vote for keeping it in English. Even in Portugal we all use the English words when talking about it."

I definitely second that approach! Although I did translate "Waypoint" to "Wegpunkt" which is more or less a literal translation but well known to and widely used by anyone familiar with GPX-Tracks in the German world...

What caught me off guard until I figured it out, was the "Enter / Center Click" which I then realized referred to the 2 different controller units (Terrain Command vs. Carpe CI). I kind of find the "Center Click" semi-intuitive... but can't think of anything better than "joystick push" which sounds crappy...
Also the "Terrain Command" wheel sensor had me stumped until I realized it was a feature of the TC Controller.... I was about to translate that to "Gelände Befehl" Radsensor which would have gotten some good laughs by the german crew.... (No, not seriously...)

One of the big challenges of UI nomenclature and their respective translations is having a conceptual continuity across menus and their translations...
